### PR TITLE
Mention the required pluginRepositories section RMB-810

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

**Review by hjiebsco — APPROVED** [review]

File: doc/upgrading.md [hunk 1]
```diff
@@ -52,6 +52,17 @@ enable generating clients:
...
+```xml
+  <pluginRepositories>
+    <pluginRepository>
+      <id>folio-nexus</id>
+      <name>FOLIO Maven repository</name>
+      <url>https://repository.folio.org/repository/maven-folio</url>
+    </pluginRepository>
+  </pluginRepositories>
+```
+
```
Comment (julianladisch) [inline doc/upgrading.md:65]: This needs to be added to the README as well: https://github.com/folio-org/raml-module-builder#step-2-include-the-jars-in-your-project-pomxml

Comment (adamdickmeiss) [reply doc/upgrading.md:65]: Ok. Will do

**Comment by adamdickmeiss** [comment]
The last commit message had bad format. I'll squash and merge, so it gets a little prettier.

(eg Git commits should have a 1 line intro, blank line and paragraph)

**Review by julianladisch — APPROVED** [review]
Thanks!

**Pull request merged**